content/docs/1.1.0/volumes-and-nodes/storage-tags.md

```diff
@@ -28,10 +28,18 @@ When creating a volume, specify the disk tag and node tag in the UI.
 
 ### Kubernetes
 
-Use Kubernetes StorageClass setting to specify tags.
+Use Kubernetes StorageClass setting to specify tags to schedule replicas only on the matching nodes.
```

Comment:
Why you changed all doc versions.. this new change is only introduced to the master/1.4.0. https://github.com/longhorn/longhorn/pull/4576/files

Reply:
I did not notice the PR wasn't backport to 1.3.x and 1.2.x, I will roll back the change of documentation.